### PR TITLE
feat(EMS-3338): no pdf - application submission - XLSX - export contract

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:

This PR updates the export contract section of the XLSX to match with the latest design.

## Resolution :heavy_check_mark:

- Add E2E test coverage for submitting single and multiple contract policies with the following export contract conditions: 
  - Fully populated with all possible answers.
  - Minimal population / least amount of possible answers.
  - Note: the existing E2E test coverage is already in great place.
- Update XLSX config indexes.
- Add unit tests for XLSX config unit tests for fully populated export contract conditions.
- Update `XLSX_ROW_INDEXES` to conditionally increment particular fields/rows depending on the submitted export contract answers.
- Update XLSX content strings.
- Update `mapApplicationToXLSX` to include a new `mapExportContract` mapping function.
- Create new XLSX mapping function for the export contract section:
  - `mapPrivateMarket`
  - `mapAgent`
  - `mapAgentCharge`
  - `mapAgentChargeAmount`
- Update `getPopulatedApplication` API helper function to create fully populated export contract data/relationships.
- Update/align API export contract types.

## Miscellaneous :heavy_plus_sign:

- Add a missing param/argument to the `completeSignInAndSubmitAnApplication` cypress command.
- Update jest test coverage config to include the `XLSX-CONFIG` directory.

### CODE REVIEW

File: src/api/types/application-types/index.ts [hunk 1]
```diff
@@ -155,36 +155,48 @@ export interface ApplicationEligibility extends Relationship {
...
-export interface ApplicationExportContractAgentServiceCharge extends Relationship {
-  agentServiceId: string;
-  percentageCharge?: string;
-  fixedSumAmount?: string;
-  fixedSumCurrencyCode?: string;
-  payableCountryCode?: string;
-  method?: string;
+export interface ApplicationExportContract extends Relationship {
+  agent: ApplicationExportContractAgent;
+  // applicationId: string;
```
Comment (ttbarnes) [inline src/api/types/application-types/index.ts:193]: TODO: need to remove commented out IDs here

File: src/api/generate-xlsx/map-application-to-XLSX/map-export-contract/map-private-market/index.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const { FIELDS } = XLSX;
+
+const {
+  PRIVATE_MARKET: { ATTEMPTED, DECLINED_DESCRIPTION },
+} = FIELD_IDS;
+
+/**
+ * mapPrivateMarket
+ * Map an application's "export contract - private market" fields into an array of objects for XLSX generation
+ * @param {Application} application
```
Comment (Zainzzkk) [inline src/api/generate-xlsx/map-application-to-XLSX/map-export-contract/map-private-market/index.ts:17]: missing params

File: src/api/helpers/get-populated-application/index.ts [hunk 3]
```diff
@@ -105,11 +105,55 @@ const getPopulatedApplication = async ({
     throw new Error(generateErrorMessage('exportContract', application.id));
   }
 
+  // TODO:
+  // TODO:
+  // TODO: move into own function
+
```
[the remark this answers is not in the document]
Comment (Zainzzkk) [reply src/api/helpers/get-populated-application/index.ts:111]: ticket number?

[the remark this answers is not in the document]
Comment (ttbarnes) [reply src/api/helpers/get-populated-application/index.ts:111]: Just spotted, sorry, going to do in this PR

[the remark this answers is not in the document]
Comment (ttbarnes) [reply src/api/helpers/get-populated-application/index.ts:111]: Actually sorry, this is a bigger piece of work and we need to do everything in `getPopulatedApplication`. Created a ticket for this EMS-3501

File: src/api/generate-xlsx/map-application-to-XLSX/map-export-contract/map-agent/map-agent-charge/map-agent-charge-amount/index.test.ts [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+      expect(result).toEqual(expected);
+    });
+  });
+
+  describe(`when the charge method is ${PERCENTAGE_CHARGE} is true`, () => {
+    const mockCharge = {
+      ...charge,
+      [FIXED_SUM_AMOUNT]: null,
+      [PERCENTAGE_CHARGE]: 10,
```
[the remark this answers is not in the document]
Comment (ttbarnes) [reply src/api/generate-xlsx/map-application-to-XLSX/map-export-contract/map-agent/map-agent-charge/map-agent-charge-amount/index.test.ts:46]: neither, just "is percentage charge", updated

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**